components/slider/Slider.razor.cs

```diff
@@ -320,17 +340,56 @@ private double RightValue
         [Parameter]
         public Action<TValue> OnChange { get; set; }
 
+        [Parameter]
+        public bool HasTooltip { get; set; } = true;
+
+        /// <summary>
+        /// Slider will pass its value to tipFormatter, and display its value in Tooltip
+        /// </summary>        
+        private bool _isTipFormatterDefault = true;
+        private Func<double, string> _tipFormatter = (d) => d.ToString(LocaleProvider.CurrentLocale.CurrentCulture);
```

Comment:
Previous `Func<string>` was replaced so `Default` values could also be shown using `TipFormatter`. Now each edge will provide its current value to and expect a string back.